tests/unit_tests/test_wandb_artifacts_full.py

```diff
@@ -193,3 +195,22 @@ def make_table():
     artifact2.add(t1, "t2")
     assert artifact2.manifest.entries["t2.table.json"].ref is not None
     run.finish()
+
+
+def test_buffer_to_disk(wandb_init, runner):
+    mani = wandb.wandb_sdk.interface.artifacts.ArtifactManifest
+    default_buffer = mani.MAX_ENTRIES_BUFFER
+    try:
```

Comment:
why not just use `with patch.object(wandb.wandb_sdk.interface.artifacts.ArtifactManifest, "MAX_ENTRIES_BUFFER", 10)`?

Reply:
Yeah, thought about that.  Was basically too lazy to look up the syntax 🤣 .  Will fix.